### PR TITLE
remove https:// from contributor url

### LINKED ISSUE #433: Remove url prefixes from credits page
AD7AA205-778C-47E1-A81A-9016EF730B7B_1_201_a

Remove `https://` from the links. Especially the two forwardslashes // look misplaced in our font

### PR DESCRIPTION
**Pull request by fremartini**
closes #433

### CODE REVIEW

File: lib/features/contributor/presentation/cubit/contributor_cubit.dart [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+part 'contributor_state.dart';
+
+class ContributorCubit extends Cubit<ContributorState> {
+  final ContributorRepository repository;
+
+  ContributorCubit({required this.repository})
+      : super(const ContributorLoaded([]));
+
+  Future<void> getContributors() async {
```
Comment (marfavi) [inline lib/features/contributor/presentation/cubit/contributor_cubit.dart:14]: Method should not be async as there are not any await statements, and no Future is returned

File: lib/features/contributor/presentation/cubit/contributor_cubit.dart [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import 'package:coffeecard/features/contributor/domain/entities/contributor.dart';
+import 'package:equatable/equatable.dart';
+
+part 'contributor_state.dart';
+
+class ContributorCubit extends Cubit<ContributorState> {
+  final ContributorRepository repository;
+
+  ContributorCubit({required this.repository})
+      : super(const ContributorLoaded([]));
```
Comment (marfavi) [inline lib/features/contributor/presentation/cubit/contributor_cubit.dart:12]: I know we discussed this once, but I'd like to contest this initialisation of the cubit state. Initially, it should *not* have the Loaded state, but rather some Initial state. Having the Loaded[] state has potential to confuse during development or debugging.

**Review by marfavi — COMMENTED** [review]
Small comments

File: lib/features/contributor/presentation/cubit/contributor_cubit.dart (hunk 2 above)
Comment (fremartini) [reply lib/features/contributor/presentation/cubit/contributor_cubit.dart:12]: what do you suggest doing instead?

Comment (marfavi) [reply lib/features/contributor/presentation/cubit/contributor_cubit.dart:12]: Add a ContributorInitial state

**Comment by fremartini** [comment]
@marfavi

**Comment by marfavi** [comment]
Will review before the meeting tomorrow

File: test/features/contributor/domain/usecase/fetch_contributors_test.dart [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+void main() {
+  late MockContributorLocalDataSource dataSource;
+  late FetchContributors usecase;
+
+  setUp(() {
+    dataSource = MockContributorLocalDataSource();
+    usecase = FetchContributors(dataSource: dataSource);
+  });
+
+  test('should call repository', () async {
```
Comment (marfavi) [inline test/features/contributor/domain/usecase/fetch_contributors_test.dart:27]: Update test name 
```suggestion
  test('should call data source', () async {
```

File: test/features/contributor/presentation/cubit/contributor_cubit_test.dart [hunk 4]
```diff
@@ -1,49 +1,62 @@
...
-      },
-      act: (cubit) => cubit.getContributors(),
-      expect: () => [
-        const ContributorLoaded(dummyContributors),
-      ],
-    );
+  group(
+    'getContributors',
+    () {
+      const tContributors = [
```
Comment (marfavi) [inline test/features/contributor/presentation/cubit/contributor_cubit_test.dart:32]: Don't use abbreviations when naming variables
```suggestion
      const testContributors = [
```

**Review by marfavi — CHANGES_REQUESTED** [review]

**Pull request merged**